doc/getting_started/getting_started.rst

```diff
@@ -30,6 +32,20 @@ using Git to clone the repository anonymously. Enter:
 You have successfully checked out a copy of the source code to your local
 machine in a ``zephyr`` folder in your home directory.
 
+The environment variable ``ZEPHYR_BASE`` must be set to the repository
```

Comment:
Let's simplify this to:

    Zephyr development tools assume the environment variable
    ``ZEPHYR_BASE`` is set to the repository path.  Here are examples
    for setting this variable on our supported development platforms: